lab-adam/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "router-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "chai": "3.5.0",
+    "gulp": "3.9.1",
+    "gulp-eslint": "3.0.1",
+    "gulp-mocha": "3.0.1",
+    "node-uuid": "1.4.7",
```

Comment:
This is a regular dependency, not a dev dependency. Your app would break without it, as your model depends upon it.